### ISSUE TITLE
.dvm layout

### ISSUE THREAD

**Issue by rgbkrk** [rgbkrk/dvm#1]
`dvm` would probably apply well to more than just `docker` for versioning go binaries, but let's tackle just `docker` for now.

```
.dvm/
  bin/
    docker/
      1.6.1/
        docker
      1.8.1/
        docker
```

Note the structure of get.docker.com:

```
Example:

https://get.docker.com/builds/Linux/x86_64/docker-1.6.1

https://get.docker.com/builds/{OS}/{ARCH}/docker-{VERSION}
```

**Comment by smashwilson** [comment]
:+1: Looks perfectly reasonable to me.

**Comment by rgbkrk** [comment]
Ok, this has been implemented in ab25d43df1eb0268c3e3e404b124d0ec882a47c4
